src/app/setup-wizard/pre-visit/company-setup/company-setup.component.html

```diff
@@ -19,6 +19,13 @@ <h5>
                         <div class="alert alert-danger form-error" *ngIf="name.touched && name.getError('required')">
                             Company Name Required
                         </div>
+                        <!-- Set Energy Unit Field -->
+                        <app-label-with-tooltip [field]="'energyUnit'" [label]="'Set Energy Unit'" [isRequired]="false"
```

Comment:
If you are going to use the label with a tooltip we will need to add a definition for the tooltip for 'energyUnit' in LabelTooltips.ts